app/src/components/EditEvent/EditEventContainer.tsx

```diff
@@ -78,7 +86,7 @@ export const EditEventContainer = () => {
   );
 
   const renderPreviewEvent = () => {
-    if (isOk(event)) {
+    if (editEventFunction && isOk(event)) {
```

Comment:
`editEventFunction` ( 🙄) Har type `false | (() => Promise<void>)` ´, men bare `false` hvis `isOk(event)` ikke gir `true`, så blir ikke testen her `isOk(event) && isOk(event)` ?

Reply:
Jo einig, men heile komponenten må refactores for å slippe den testen. Det blir med ny side for preview!